interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -86,7 +86,7 @@ namespace {
     return cling::Interpreter::kExeSuccess;
   }
 
-  static bool isPracticallyEmptyModule(const llvm::Module* M) {
+  static bool isPracticallyEmptyModule(std::shared_ptr<const llvm::Module> M) {
```

Comment:
Should just take a `Module*`; I don't see that ownership should be involved here?